src/core/core.scale.js

```diff
@@ -782,7 +797,7 @@ var Scale = Element.extend({
 		var cos = Math.abs(Math.cos(rot));
 		var sin = Math.abs(Math.sin(rot));
 
-		var labelSizes = me._labelSizes;
+		var labelSizes = me._getLabelSizes();
```

Comment:
I dislike added function call for each value.
If we agree on #6343, I'd investigate computing labelSizes in `_configure`.

Reply:
There's negligible performance impact because `_tickSize` is called only once. Calling in `_configure` might be rather tricky because it depends on `convertTicksToLabels` having been called already